plans/util/init_db_server.pp

```diff
@@ -0,0 +1,91 @@
+plan peadm::util::init_db_server(
```

Comment:
puppet-strings documentation is missing :(

Reply:
I think this should maybe be private. I will update documentation for this.